admin_manual/index.rst

```diff
@@ -1,7 +1,3 @@
-============
-Introduction
-============
-
 Introduction
```

Comment:
you are the expert :) I've seen the title being twice there, but I did not know how to make a pull request that removes either the one in the top or the one under it.